mantidproject/thirdparty-msvc2015
# Kafka Windows dependencies.

Fixes #14
Dependencies for Kafka live listener added to third party repo.

To test:

* Run `[PathToThirdPartyFolder]/build-scripts/build-librdkafka`
* check the `[PathToThirdPartyFolder]/bin` and `[PathToThirdPartyFolder]/lib` for the following files:
```
librdkafka.dll/pdb
librdkafka_D.dll/pdb
librdkafkacpp.dll/pdb
librdkafkacpp_D.dll/pdb
librdkafka.lib/librdkafka_D.lib
librdkafkacpp.lib/librdkafkacpp_D.lib
```

Further tests to ensure functionality with the mantid codebase can be found [here](https://github.com/ScreamingUdder/mantid/pull/5)
